### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/gatherer.git gatherer
    cd gatherer
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/dependabot.yml README.md bump_version.sh 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -4,34 +4,38 @@ FROM python:3.11.0-slim-bullseye
 
 ARG VERSION
```
Comment (mcdonnnj) [inline Dockerfile]: This argument is not used in this configuration.

File: Dockerfile [hunk 2]
```diff
@@ -86,14 +92,13 @@ RUN apt-get remove --quiet --quiet $INSTALL_DEPS
 RUN apt-get --quiet --quiet clean
 RUN rm -rf /var/lib/apt/lists/*
```
Comment (mcdonnnj) [inline Dockerfile]:
```suggestion
RUN apt-get --quiet --quiet clean
    && rm --recursive --force /var/lib/apt/lists/*
```

File: Dockerfile [hunk 3]
```diff
@@ -86,14 +92,13 @@ RUN apt-get remove --quiet --quiet $INSTALL_DEPS
...
 # Setup working directory and entrypoint
-###
-
+#
 # Put this just before we change users because the copy (and every
 # step after it) will always be rerun by docker, but we need to be
 # root for the chown command.
+###
 COPY src ${CISA_HOME}
 RUN chown -R ${CISA_USER}:${CISA_GROUP} ${CISA_HOME}
```
Comment (mcdonnnj) [inline Dockerfile]:
```suggestion
RUN chown --recursive ${CISA_USER}:${CISA_GROUP} ${CISA_HOME}
```

File: .github/labels.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- color: "ce099a"
+  description: This pull request is ready to merge during the next Lineage Kraken release
+  name: kraken 🐙
+- color: "a4fc5d"
+  description: This issue or pull request requires further information
+  name: need info
+- color: "fcdb45"
+  description: This pull request is awaiting an action or decision to move forward
+  name: on hold
+- color: "ef476c"
```
Comment (mcdonnnj) [inline .github/labels.yml:59]:
```suggestion
- color: "3772a4"
  description: Pull requests that update Python code
  name: python
- color: "ef476c"
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Please change https://github.com/cisagov/gatherer/blob/cb66bfdaca6c539b65f8c71d381f77c261c6c1a7/Dockerfile#L106 to `run`.

**Comment by jsf9k** [comment]
> Please change
> 
> https://github.com/cisagov/gatherer/blob/cb66bfdaca6c539b65f8c71d381f77c261c6c1a7/Dockerfile#L106
> 
> to `run`.

Please see commit e974771.

File: Dockerfile (hunk 1 above)
Comment (jsf9k) [reply Dockerfile]: Please see commit 3ed6100.

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**